### DISCUSSION TITLE
add syslog notifications

### DISCUSSION THREAD

**Discussion by wallacebrf** [007revad/Synology_app_mover#32] [Ideas]
you could add syslog notifications using

`/usr/syno/bin/synologset1 sys err 0x11100000 "YOUR TEXT HERE"`

i forget what the "0x11100000" does, but the "err" is for a syslog error, but you should be able to do warnings and info submissions as well.

**Comment by 007revad** [comment]
Yep. 7 of my scripts use synologset1 to log auto update errors. 

You can log info, warn and err. The 0x11100000 is a key name from `/usr/syno/synosdk/texts/enu/events` and it's the one with a value that allows logging your own message.

There are 7 others (at the top of the file) that allow logging your own messages but they appear to have been left there by a Synology developer so we can't trust they'll still exist in future DSM updates.
